src/Serilog.AspNetCore/AspNetCore/SerilogLoggerFactory.cs

```diff
@@ -18,26 +18,49 @@
 
 namespace Serilog.AspNetCore
 {
-    class SerilogLoggerFactory : ILoggerFactory
+	/// <summary>
+	/// Implements Microsoft's ILoggerFactory so that we can inject Serilog Logger.
```

Comment:
Instead of `Microsoft's ILoggerFactory` we could use `<see cref="ILoggerFactory"/>`